Firebase/InstanceID/FIRIMessageCode.h

```diff
@@ -66,6 +66,7 @@ typedef NS_ENUM(NSInteger, FIRInstanceIDMessageCode) {
   kFIRInstanceIDMessageCodeService005 = 7005,
   kFIRInstanceIDMessageCodeService006 = 7006,
   kFIRIntsanceIDInvalidNetworkSession = 7007,
+  kFIRIntsanceIDInvalidSettingResponse = 7008,
```

Comment:
Spelling here and also kFIRIntsanceIDInvalidNetworkSession above.

Reply:
Oops. Done.